ethereum/executionclient/src/main/java/tech/pegasys/teku/ethereum/executionclient/auth/JwtConfig.java

```diff
@@ -32,21 +32,30 @@ public class JwtConfig {
   public static final long EXPIRES_IN_SECONDS = TOLERANCE_IN_SECONDS - 2;
 
   private final SecretKey key;
+  private final Optional<String> secretId;
```

Comment:
I think the variable name should be `id` and not `secretId`. Or `idClaim` if you want a longer\clearer one.

Reply:
and infact I think that the param itself should be `--ee-jwt-id` (or with a claim too)

Reply:
I think `id` is too generic. I like `claimId` :)

Reply:
Agree, I like `claimId` as well. Thanks. Pushed change to reflect that.